scripts/release/publish.ts

```diff
@@ -124,7 +124,7 @@ const publishAllPackages = async ({
   dryRun?: boolean;
 }) => {
   console.log(`📦 Publishing all packages...`);
-  const command = `yarn workspaces foreach --parallel --no-private --verbose npm publish --tolerate-republish --tag ${tag}`;
+  const command = `yarn workspaces foreach --all --parallel --no-private --verbose npm publish --tolerate-republish --tag ${tag}`;
```

Comment:
The previous default in v3 was `worktree` instead of `all`. Can you explain, why we use `all` now?

Reply:
TBH I didn't understand the difference between "worktree" and "all", especially in our scenario, so I just went with what they have in the example in the docs.

Reply:
I guess that it is context-specific. If `process.cwd()` would return a subfolder, yarn would only consider all projects in that subfolder when `--worktree` is used. With `--all` yarn will consider all projects defined in the root package.json

Reply:
I see that makes sense, so it doesn't make a difference for us.